Adding a sentence to explain the interchangeable terms (#16055)
hashicorp/consul: website/content/docs/architecture/improving-consul-resilience.mdx

  Fault tolerance is a system's ability to operate without interruption despite component failure. Learn how a set of Consul servers provide fault tolerance through use of a quorum, and how to further improve control plane resilience through use of infrastructure zones and Enterprise redundancy zones.
---

# Fault (Failure) Tolerance

> Fault tolerance and failure tolerance are often used interchangably. Fault tolerance is the ability of a system to continue operating without interruption

Reviewer: I agree with the need to state that fault tolerance and failure tolerance are interchangeable terms. However, this is an auxiliary detail, not the most important thing about this topic, so it shouldn't be the only thing contained in the first sentence.

Maybe something like this?
```suggestion
Fault tolerance, also known as failure tolerance, is the ability of a system to continue operating without interruption
```

@im2nguyen : What do you think?
